x/committee/types/msg.go

```diff
@@ -62,9 +62,10 @@ func (msg MsgSubmitProposal) GetSigners() []sdk.AccAddress {
 type VoteType int
 
 const (
-	Yes     VoteType = iota // 0
-	No      VoteType = iota // 1
-	Abstain VoteType = iota // 2
+	NullVoteType VoteType = iota // 0
+	Yes          VoteType = iota // 1
+	No           VoteType = iota // 2
+	Abstain      VoteType = iota // 3
 )
 
```

Comment:
nit / suggestion:
```suggestion
func (vt VoteType) Validate() error {
	if vt < 0 || vt > 3 { 
		return fmt.Errorf("invalid vote type: %d", vt)
	}
	return nil
}
```
A validate function like this would avoid duplicating the validate code between `msg.ValidateBasic` and `vote.Validate`. If a new vote type is added, there'd be only one piece of code to update.

Reply:
https://github.com/Kava-Labs/kava/pull/902/commits/23bba6964e52dd2095212ade22c4a7adac3f69a1